app/views/cars/_car.html.slim

```diff
@@ -17,3 +17,6 @@ div id="#{dom_id car}"
   p
     strong License plate:
     =< car.license_plate
+  p
+    strong User ID:
+    =< car.user_id
```

Comment:
This abstraction break is unnecessary, a user shouldn't know or care what id they have

Reply:
These are old views, which you are using form the broken main. I just merge main with home_page so u can git clone it now .

Reply:
It was like that in add_cars branch.